<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContainerForm</class>
 <widget class="QWidget" name="ContainerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnArray">
       <property name="text">
        <string>Array</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnVector">
       <property name="text">
        <string>Vector</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeque">
       <property name="text">
        <string>Deque</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFList">
       <property name="text">
        <string>Forward List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnList">
       <property name="text">
        <string>List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStack">
       <property name="text">
        <string>Stack</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQueue">
       <property name="text">
        <string>Queue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPQueue">
       <property name="text">
        <string>Priority Queue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSet">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMset">
       <property name="text">
        <string>Multiset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMap">
       <property name="text">
        <string>Map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMmap">
       <property name="text">
        <string>Multimap</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUnSet">
       <property name="text">
        <string>Unordered Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUnMset">
       <property name="text">
        <string>Unordered Multiset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUnMap">
       <property name="text">
        <string>Unordered Map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUnMmap">
       <property name="text">
        <string>Unordered Multimap</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget_2"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btnArray_2">
       <property name="text">
        <string>Array</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnVector_2">
       <property name="text">
        <string>Vector</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeque_2">
       <property name="text">
        <string>Deque</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFList_2">
       <property name="text">
        <string>Forward List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnList_2">
       <property name="text">
        <string>List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStack_2">
       <property name="text">
        <string>Stack</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQueue_2">
       <property name="text">
        <string>Queue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPQueue_2">
       <property name="text">
        <string>Priority Queue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSet_2">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMset_2">
       <property name="text">
        <string>Multiset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMap_2">
       <property name="text">
        <string>Map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMmap_2">
       <property name="text">
        <string>Multimap</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUnSet_2">
       <property name="text">
        <string>Unordered Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUnMset_2">
       <property name="text">
        <string>Unordered Multiset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUnMap_2">
       <property name="text">
        <string>Unordered Map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUnMmap_2">
       <property name="text">
        <string>Unordered Multimap</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
